templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml

```diff
@@ -21,6 +21,7 @@ contents:
     # extract etcdctl binary from container image
     dl_etcdctl() {
       local etcdimg="{{.Images.etcdKey}}"
+      podman image pull "${etcdimg}" --authfile=/var/lib/kubelet/config.json
       local etcdctr=$(podman create "${etcdimg}" --authfile=/var/lib/kubelet/config.json)
```

Comment:
podman latest shows pull missing as default. Just making sure this is needed maybe it is a podman versioning issue.

```
--pull string                              Pull image before creating ("always"|"missing"|"never") (default "missing")
```

Reply:
This could be a bug in `podman` but we need this change at least as a workaround.